.github/workflows/exit-pre-mode-on-comment.yml

```
@@ -0,0 +1,44 @@
+name: Exit pre-release changesets
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  exit-pre-release:
+    name: Exit changesets pre-release mode when !prod-ready comment created
+    if: ${{ github.event.comment.body == '!prod-ready' && github.event.issue.pull_request }}
```

Can we do it in the similar fashion as here?
```suggestion
    if: ${{ contains(github.event.comment.body, '!prod-ready') && github.event.issue.pull_request }}
```

---

At least it will be unified